### PR TITLE
Feature/erssup 60942

### PR DESCRIPTION
**Pull request by nhsd-angel-pastor**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/components/schemas/responses/TooManyRequests.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  This protects our service against excessive use and denial-of-service (DoS) attacks, and is also to encourage you to use our APIs efficiently.
+  Our standard rate limit for the production environment is 10 requests per second per application.
+  APIs associated with the "Binary" resource are further restricted to 2 requests per second per application.
+  If you go over the rate limit you'll receive a response with an HTTP status of 429 (Too Many Requests).
+  Our path-to-live environments have very low rate limits.
+  They are for functional testing only - you should not use them for [[HYPERLINK_PERFORMANCE_TESTING]].
+  If you have problems with rate limits, [[HYPERLINK_CONTACT_US]] to discuss your application design and volumetrics,
+  and to see whether it's appropriate to raise your rate limit.
+  If a 429 response is received, indicating a request has been rate limited, then you must back off and retry with exponentially increasing periods between requests.
+  Once a request is successful again, then you may continue to send requests inline with the documented rate limits.
```
Comment (nhsd-jack-wainwright) [inline specification/components/schemas/responses/TooManyRequests.yaml:12]: I think this description will need to be updated once the rate limits are agreed for the patient care APIs.

**Review by nhsd-jack-wainwright — COMMENTED** [review]
Looks good to me 🙂, just a few minor comments.

**Review by nhsd-radoslav-radev — APPROVED** [review]
If Jack is happy with this we can simply merge it now.

**Pull request merged**